File: 7-kyu/jumping-number-(special-numbers-series-#4)/python/solution.py
def jumping_number(number):
    number_lst = list(map(int, str(number)))
    number_zip = zip(number_lst, number_lst[1:])
    return 'Jumping!!' if all(abs(item[0] - item[1]) == 1 for item in number_zip) else 'Not!!'

File: 7-kyu/jumping-number-(special-numbers-series-#4)/swift/solution.swift
func jumpingNumber(_ number: Int) -> String {
    guard number > 9 else { return "Jumping!!" }
    let numberArr = String(number).compactMap { Int(String($0)) }
    let numberArrZipped = zip(numberArr, numberArr[1...])
    for (x, y) in numberArrZipped {
        if abs(x - y) != 1 { return "Not!!" }
    }
    return "Jumping!!"
}